spring-kafka/src/main/java/org/springframework/kafka/core/KafkaOperations.java

```diff
@@ -64,6 +64,16 @@
 	ListenableFuture<SendResult<K, V>> sendDefault(int partition, K key, V data);
 
 	/**
+	 * Send the data to the default topic with the provided key and partition.
+	 * @param partition the partition.
+	 * @param timestamp the timestamp of the record.
+	 * @param key the key.
+	 * @param data the data.
+	 * @return a Future for the {@link SendResult}.
+	 */
+	ListenableFuture<SendResult<K, V>> sendDefault(int partition, Long timestamp, K key, V data);
```

Comment:
Your name to the `@author` list and copyright.

The general JavaDoc on the method should mention `timestamp` as well.

I see from where you have picked up the `Long` type wrapper, not primitive.
And it looks for me to be consistent we should do the same for `partition` argument.
But that will be already breaking change if we are going to backport.

Although, we won't backport this interface changes any way.
WDYT?